### PR TITLE
1319 - Added ExtractRecycleBin module to RA

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import static org.sleuthkit.datamodel.BlackboardAttribute.ATTRIBUTE_TYPE.TSK_DATETIME_DELETED;
+import static org.sleuthkit.datamodel.BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PATH;
+import org.sleuthkit.datamodel.Content;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+import org.sleuthkit.datamodel.TskDataException;
+
+/**
+ * This module is based on the RecycleBin python module from Mark Mackinnon.
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: McKinnon not Mackinnon

Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: Sorry about that!

**Comment by rcordovano** [comment]
@markmckinnon, please test this PR in addition to reviewing the code, thanks!

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        this.moduleName = Bundle.ExtractRecycleBin_module_name();
+    }
+
+    @Override
+    void process(Content dataSource, IngestJobContext context, DataSourceIngestModuleProgress progressBar) {
+        FileManager fileManager = Case.getCurrentCase().getServices().getFileManager();
+        String tempDirPath = RAImageIngestModule.getRATempPath(Case.getCurrentCase(), "recyclebin"); //NON-NLS
+        SleuthkitCase skCase = Case.getCurrentCase().getSleuthkitCase();
+
+        // At this time it was decided tjat we would not include TSK_RECYCLE_BIN
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: that instead of tjat

Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java]: Fixed in next PR for the user name.

**Comment by kellykelly3** [comment]
@rcordovano The codacy issues are fixed in https://github.com/sleuthkit/autopsy/pull/5122

**Pull request merged**